index.html

```diff
@@ -201,7 +201,7 @@ <h2>Scope</h2>
                     maintain those specifications, ensuring that no
                     <a href="https://www.w3.org/policies/process/20231103/#class-4">Class 4
                         changes</a> are introduced except as needed to address any serious security
-                    issues that arise.
+                    issues that arise or to support the new Recommendations produced by the group.
```

Comment:
I am o.k. with this addition, which is necessary if that restriction on class 4 stays in the document. However, see issue #7 which may make this entry obsolete.

(But it is better to merge this for now.)